### PR TITLE
de: Translated Chapter 1

### PR DESCRIPTION
**Pull request by Throvn**
I'm sorry for the confusion but it is best to revert back to the rust book syntax when translating important rust vocabulary ("German (english)" instead of "English (german)").
The reason behind that is that otherwise you would have to conjugate the english word which would turn it into "Denglish". 

Also the reading flow is way better:
- "Für den Async-spezifischen Teil an Tag 4 benötigst du eine frisch erstellte Kiste (crate)." vs.
- "Für den Async-spezifischen Teil an Tag 4 benötigst du eine frisch erstellte Crate (Kiste)."

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -1352,6 +1350,30 @@ msgid ""
 "\n"
 "1. Prepare anything you need to have available for the afternoon of day 4."
 msgstr ""
+"1. Setze dich mit dem Kursmaterial auseinander. Wir haben Sprechernotizen hinzugefügt um "
```
Comment (ronaldfw) [inline po/de.po:1353]:
```suggestion
"1. Setze dich mit dem Kursmaterial auseinander. Wir haben Sprechernotizen hinzugefügt, um "
```

File: po/de.po [hunk 2]
```diff
@@ -1352,6 +1350,30 @@ msgid ""
...
+"Beim Vorstellen kannst du die Sprechernotizen in einem Seitenfenster öffnen (klick dazu auf den "
+"kleinen Pfeil neben \"Speaker Notes\"). Dadurch hast du eine übersichtliche Oberfläche wenn du den "
+"Kurs vorstellst.\n"
+"1. Am Nachmittag des vierten Tages kannst du ein Thema frei wählen. Du kannst entweder ein Thema "
+"nach Schwierigkeitsgrad wählen, oder aber ein Thema was zu deiner Zuhörerschaft passt.\n"
+"1. Lege im Vorhinein die Kurszeiten fest. Der Kurs ist umfangreich, daher empfehlen wir vier Tage "
+"in einem Zeitraum von zwei Wochen. Kursteilnehmer meinten, dass Pausen zwischen den Tagen sinnvoll "
+"seien um die Menge an Informationen zu verarbeiten.\n"
+"1. Finde einen Raum der groß genug für alle anwesenden Teilnehmer ist. Wir empfehlen eine "
+"Teilnehmeranzahl von 15 bis 20 Leuten. Das schafft ein gutes Klima um Fragen zu stellen und es ist "
```
Comment (ronaldfw) [inline po/de.po:1364]:
```suggestion
"Teilnehmeranzahl von 15 bis 20 Leuten. Das schafft ein gutes Klima, um Fragen zu stellen, und es ist "
```

File: po/de.po [hunk 3]
```diff
@@ -1352,6 +1350,30 @@ msgid ""
...
+"möglich alle Fragen zeitnah zu beantworten. Stelle auch sicher, dass der Raum _Tische_ mit Stühlen "
+"für die Teilnehmer und dich hat. Ihr werdet nämlich eure Laptops benötigen. Um genau zu sein wirst "
+"du viel vorprogrammieren.\n"
+"1. Sei schon etwas vor dem ersten Termin da um alles vorzubereiten. Wir schlagen vor, dass du "
```
Comment (ronaldfw) [inline po/de.po:1368]:
```suggestion
"1. Sei schon etwas vor dem ersten Termin da, um alles vorzubereiten. Wir schlagen vor, dass du "
```

File: po/de.po [hunk 4]
```diff
@@ -1352,6 +1350,30 @@ msgid ""
...
+"`mdbook serve` direkt präsentierst (siehe [Installationsanleitung][3]). Somit entstehen keine "
+"Störungen oder Unklarheiten bei dem Wechseln von Seiten und falls du oder die Teilnehmer "
+"Rechtschreibfehler finden können diese sofort behoben werden.\n"
+"1. Gib den Kursteilnehmern Zeit die Aufgaben selbst oder in kleinen Gruppen zu lösen. Vergiss "
+"nicht ab und an zu fragen ob jemand Hilfe benötigt. Wenn mehrere Teilnehmer das gleiche Problem "
+"haben, spreche es vor allen Teilnehmern an und zeige einen möglichen Lösungsweg auf. Z. B. "
+"Könntest du zeigen wo man die relevante Information in der Standardbibliothek findet.\n"
```
Comment (ronaldfw) [inline po/de.po:1375]:
```suggestion
"könntest du zeigen wo man die relevante Information in der Standardbibliothek findet.\n"
```

File: po/de.po [hunk 5]
```diff
@@ -1407,68 +1428,73 @@ msgstr ""
...
+"Der Nachmittag des vierten Tages sollte ein von dir ausgewähltes Thema behandeln.\n"
+"Erwähne das ausgewählte Thema in der Kursankündigung, sodass Teilnehmer wissen was auf sie zukommt."
 
 #: src/running-the-course/day-4.md:9
 msgid ""
 "This phase of the course is a chance for participants to see Rust in action on a\n"
 "codebase they might be familiar with. You can choose from the topics already\n"
 "defined here, or plan your own."
 msgstr ""
+"Dieser Abschnitt des Kurses ermöglicht es Teilnehmern Rust in einer Codebasis zu sehen die "
```
Comment (ronaldfw) [inline po/de.po:1448]:
```suggestion
"Dieser Abschnitt des Kurses ermöglicht es Teilnehmern Rust in einer Codebasis zu sehen die sie"
```

File: po/de.po [hunk 6]
```diff
@@ -1407,68 +1428,73 @@ msgstr ""
...
 
 #: src/running-the-course/day-4.md:28
 msgid ""
 "If you chose Async for Day 4 afternoon, you will need a fresh crate set up and\n"
 "the dependencies downloaded and ready to go. You can then copy/paste the\n"
 "examples into `src/main.rs` to experiment with them."
 msgstr ""
+"Für den Async-spezifischen Teil an Tag 4 benötigst du eine frisch erstellte Kiste (crate). "
+"Ausserdem sollten die Abhängigkeiten (dependencies) bereits heruntergeladen sein. Danach können "
+"die Beispiele in `src/main.rs` eingefügt werden um mit ihnen zu experimentieren."
```
Comment (ronaldfw) [inline po/de.po:1497]:
```suggestion
"die Beispiele in `src/main.rs` eingefügt werden, um mit ihnen zu experimentieren."
```

File: po/de.po [hunk 7]
```diff
@@ -1501,31 +1527,35 @@ msgstr ""
...
 "* [Brazilian Portuguese][pt-BR] by [@rastringer] and [@hugojacob].\n"
 "* [Korean][ko] by [@keispace], [@jiyongp] and [@jooyunghan]."
 msgstr ""
+"* [Brasilianisches Portugiesisch][pt-BR] von [@rastringer] und [@hugojacob].\n"
+"* [Koreanisch][ko] von [@keispace], [@jiyongp] und [@jooyunghan]."
 
 #: src/running-the-course/translations.md:9
 msgid "Use the language picker in the top-right corner to switch between languages."
-msgstr ""
+msgstr "Benutze die Sprachenauswahl oben rechts um die Sprache zu wechseln."
```
Comment (ronaldfw) [inline po/de.po:1549]:
```suggestion
msgstr "Benutze die Sprachenauswahl oben rechts, um die Sprache zu wechseln."
```

**Review by ronaldfw — CHANGES_REQUESTED** [review]
LGTM, only some minor nits

**Review by mgeisler — APPROVED** [review]
Thanks @Throvn for applying the changes requested by @ronaldfw. Let's merge this now so that it becomes easier to iterate on.

**Pull request merged**